atomicfu-transformer/src/main/kotlin/kotlinx/atomicfu/transformer/AtomicFUTransformerBase.kt

```diff
@@ -0,0 +1,66 @@
+package kotlinx.atomicfu.transformer
+
+import org.objectweb.asm.tree.AbstractInsnNode
+import org.objectweb.asm.tree.InsnList
+import java.io.File
+import org.slf4j.Logger
+
+abstract class AtomicFUTransformerBase(
+    var inputDir: File,
+    var outputDir: File
+) {
+    protected operator fun File.div(child: String) =
+        File(this, child)
+
+    protected fun File.toOutputFile(): File =
+        outputDir / relativeTo(inputDir).toString()
+
+    protected abstract var logger: Logger
```

Comment:
Don't make it `abstract` and it should be `val`, not `var`. Just initialize it as `protected val logger = LoggerFactory.getLogger(this::class.java)`